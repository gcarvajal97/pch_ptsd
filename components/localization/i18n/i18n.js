export const fallback = "en";
 
export const supportedLocales = {
    en: {
        name: "English",
        translationFileLoader: () => require('../lang/en.json'),
        // en is default locale in Moment
        momentLocaleLoader: () => Promise.resolve(),
    },
    es: {
        name: "EspaÃ±ol",
        translationFileLoader: () => require('../lang/es.json'),
        momentLocaleLoader: () => import('moment/locale/es'),
    },
};
 
export const defaultNamespace = "LearnMore";
 
export const namespaces = [
    "LearnMore"
];