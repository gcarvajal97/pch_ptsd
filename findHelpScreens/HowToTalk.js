import i18n from 'i18n-js';
import React, { Component } from "react";
import { View, StyleSheet, Text, ScrollView } from "react-native";
import { Video } from 'expo-av';
import translate, { getVideo } from "../components/translateService";
import colors from '../colors';

const videos = {
    en: require("../assets/videos/vidHowToTalk.mp4"),
    es: require("../assets/videos/vidHowToTalkEs.mp4"),
};

class HowToTalk extends Component {

    // Gives the NavBar a title for this specific screen
    static navigationOptions = {
        headerTitle: translate('howToTalk.header')
    }

    render() {
        return (
            // renders the learn more screen
            <View style={styles.container}>
                <Text style={styles.title}>
                    {translate('howToTalk.video_title')}
                </Text>
                <Video
                    source={getVideo(videos)}
                    rate={1.0}
                    volume={1.0}
                    resizeMode={Video.RESIZE_MODE_STRETCH}
                    resizeMode={Video.RESIZE_MODE_COVER}
                    shouldPlay
                    useNativeControls
                    style={{ height: 204, width: '100%', maxHeight: 254, borderColor: colors.accent, borderWidth: 14, borderTopWidth: 0, borderBottomWidth: 0, marginBottom:-1}}
                    accessible
                    accessibilityLabel={translate('howToTalk.accessibilityLabel')}
                    accessibilityHint={translate('howToTalk.accessibilityHint')}
                />
                <Text style={styles.subTitle}>
                <Text style={{ fontFamily: "avenir-heavy" }}>{translate('howToTalk.video_subtitle_1')}</Text>{translate('howToTalk.video_subtitle_2')}</Text>

                <ScrollView>

                    <Text style={styles.paragraph}>{translate('howToTalk.subtitle')}</Text>

                    <Text style={styles.paragraphTitle}>{translate('howToTalk.title_1')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_11')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_12')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_12_a')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_12_b')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_13')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_14')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_14_a')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_14_b')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_14_c')}</Text>

                    <Text style={styles.paragraphTitle}>{translate('howToTalk.title_2')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_21')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_21')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_22')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_22')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_23')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_23')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_24')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_24_a')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_24_b')}</Text>

                    <Text style={styles.paragraphTitle}>{translate('howToTalk.title_3')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_31')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_32')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_33')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_34')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_35')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_35_a')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_35_b')}</Text>

                    <Text style={styles.paragraphTitle}>{translate('howToTalk.title_4')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_41')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_41')}</Text>
                    <Text style={styles.bullet}>{translate('howToTalk.tip_42')}</Text>
                    <Text style={styles.subBullet}>{translate('howToTalk.subtip_42')}</Text>

                    <Text style={{ marginBottom:1 }}></Text>

                </ScrollView>
            </View>
        );
    }
}
export default HowToTalk;

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        margin: 10,
        backgroundColor: colors.primary.white,
        shadowColor: colors.primary.black,
        shadowOffset: {
            width: 0,
            height: 2,
        },
        shadowOpacity: 0.23,
        shadowRadius: 2.62,
        elevation: 4,
    },
    title: {
        backgroundColor: colors.accent,
        color: colors.primary.white,
        fontFamily: "avenir-heavy",
        fontSize: 22,
        textAlign: 'center',
        alignSelf: 'stretch'
    },
    subTitle: {
        backgroundColor: colors.accent,
        color: colors.primary.white,
        alignSelf: 'stretch',
        paddingLeft: 15,
        paddingRight:15,
        fontSize: 15,
        fontFamily: "avenir-medium",
        borderBottomWidth:2,
        borderBottomColor: colors.accent,
    },
    paragraphTitle: {
        fontFamily: "avenir-heavy",
        fontSize:20,
        alignSelf:'stretch',
        textAlign:'left',
        marginLeft:5,
        marginTop:4,
        marginBottom:3,
        color: colors.accent
    },
    bullet: {
        marginVertical:1,
        marginHorizontal:18,
        textAlign:'left',
        alignSelf:'stretch',
        fontFamily: "avenir-heavy",
        fontSize:14
    },
    subBullet: {
      marginVertical:1,
      marginHorizontal:30,
      textAlign:'left',
      alignSelf:'stretch',
      fontFamily: "avenir-medium",
      fontSize:14,
    },
    paragraph: {
        alignSelf:"center",
        paddingVertical:1,
        paddingHorizontal:5,
        fontSize: 14,
        marginTop: 5,
        fontFamily: "avenir-medium"
    }
});