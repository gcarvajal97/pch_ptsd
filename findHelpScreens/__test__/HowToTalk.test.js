import i18n from "i18n-js";
import React from "react";
import renderer from "react-test-renderer";
import HowToTalk from '../HowToTalk';


describe("HowToTalk", () => {
  let wrapper;
    beforeEach(() => {
        i18n.locale = "en";
    });

    test("renders correctly", () => {
        const tree = renderer.create(<HowToTalk/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

});