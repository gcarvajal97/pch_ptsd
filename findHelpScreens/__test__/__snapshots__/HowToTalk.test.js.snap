// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToTalk renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#2089DC",
        "color": "white",
        "fontSize": 22,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    How To Talk With Your Child
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#2089DC",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video on how how to talk with your child"
      accessibilityLabel="Video on how how to talk with your child"
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#2089DC",
        "borderBottomColor": "#2089DC",
        "borderBottomWidth": 2,
        "color": "white",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      It's not always easy to know how to talk with your child. 
    </Text>
    Here are some tips to help.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        After something scary like an injury, it is not always easy to know how to talk with your child about it, and how best to help.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Here are some things to help you when talking about trauma with your child:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Be available to listen.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Provide opportunities to talk.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Look for opportunities to hear how your child is doing.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Choose a time and a place where you both feel comfortable - think about when and where your child tends to talk most openly.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Ask your child direct questions. 'How are you doing with all of this?'
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Listen carefully to what your child has to say to you.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Wait until your child finishes before jumping in.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Repeat back what you heard in your own words to be sure you understood.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Try to hold off giving advice until you have a good understanding of what your child is going through. What's been most upsetting to her? How is she feeling now?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        What do I say?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Just listening is helpful.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - You can give your child a chance to work out his thoughts and feelings by saying them out loud to you.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Answer your childâ€™s questions as honestly as you can.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Keep in mind their age and what they are able to understand.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Help your child find words for strong feelings.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Describing and understanding his reactions is a first step to dealing with them.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Share your own experiences within limits.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Be a model for using good coping skills.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Remember not to burden your child with your own worries. Turn to other adults you trust if you need a listening ear.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        What if my child does not want to talk?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Be patient. Your child may not be ready to talk at the same time you are.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Do not force the issue. Do not try to force your child to talk. Some people like to think about things for a while first.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Encourage other ways of expressing thoughts and feelings. Your child can draw a picture, write a story, or keep a diary or journal.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Help your child stay connected with friends and adults you both trust, so you will know they have people to talk with when they are ready.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        5. Some reactions are not said in words.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Even without words, you can try to be aware of your childâ€™s reactions.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Think about how you can usually tell how your child is feeling - you might see changes in behavior, facial expressions, or body language.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Things to remember:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Different people have different reactions.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - As you think about how to help your child, try to separate what you are feeling from what your child is experiencing and what they need.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Make sure your own feelings do not get in the way.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        If talking with your child is upsetting to you, do your best to stay calm and collected during the conversation and be sure to get support for yourself.
      </Text>
      <Text
        style={
          Object {
            "marginBottom": 1,
          }
        }
      />
    </View>
  </RCTScrollView>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#2089DC",
        "color": "white",
        "fontSize": 22,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    How To Talk With Your Child
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#2089DC",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video on how how to talk with your child"
      accessibilityLabel="Video on how how to talk with your child"
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#2089DC",
        "borderBottomColor": "#2089DC",
        "borderBottomWidth": 2,
        "color": "white",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      It's not always easy to know how to talk with your child. 
    </Text>
    Here are some tips to help.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        After something scary like an injury, it is not always easy to know how to talk with your child about it, and how best to help.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Here are some things to help you when talking about trauma with your child:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Be available to listen.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Provide opportunities to talk.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Look for opportunities to hear how your child is doing.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Choose a time and a place where you both feel comfortable - think about when and where your child tends to talk most openly.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Ask your child direct questions. 'How are you doing with all of this?'
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Listen carefully to what your child has to say to you.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Wait until your child finishes before jumping in.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Repeat back what you heard in your own words to be sure you understood.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Try to hold off giving advice until you have a good understanding of what your child is going through. What's been most upsetting to her? How is she feeling now?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        What do I say?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Just listening is helpful.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - You can give your child a chance to work out his thoughts and feelings by saying them out loud to you.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Answer your childâ€™s questions as honestly as you can.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Keep in mind their age and what they are able to understand.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Help your child find words for strong feelings.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Describing and understanding his reactions is a first step to dealing with them.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Share your own experiences within limits.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Be a model for using good coping skills.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Remember not to burden your child with your own worries. Turn to other adults you trust if you need a listening ear.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        What if my child does not want to talk?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Be patient. Your child may not be ready to talk at the same time you are.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Do not force the issue. Do not try to force your child to talk. Some people like to think about things for a while first.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        3. Encourage other ways of expressing thoughts and feelings. Your child can draw a picture, write a story, or keep a diary or journal.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        4. Help your child stay connected with friends and adults you both trust, so you will know they have people to talk with when they are ready.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        5. Some reactions are not said in words.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Even without words, you can try to be aware of your childâ€™s reactions.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Think about how you can usually tell how your child is feeling - you might see changes in behavior, facial expressions, or body language.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#2089DC",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Things to remember:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        1. Different people have different reactions.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - As you think about how to help your child, try to separate what you are feeling from what your child is experiencing and what they need.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "fontWeight": "bold",
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        2. Make sure your own feelings do not get in the way.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontSize": 14,
            "marginHorizontal": 30,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        If talking with your child is upsetting to you, do your best to stay calm and collected during the conversation and be sure to get support for yourself.
      </Text>
      <Text
        style={
          Object {
            "marginBottom": 1,
          }
        }
      />
    </View>
  </RCTScrollView>
</View>
`;
